docker/all-in-one/entrypoint.sh

```diff
@@ -51,13 +53,57 @@ function enable_swap {
 }
 
 function create_lsn_checkpoint_file {
-  if [ ! -f "${DATA_VOLUME_MOUNTPOINT}/latest-lsn-checkpoint" ]; then
-    echo "0/0" > "${DATA_VOLUME_MOUNTPOINT}/latest-lsn-checkpoint"
-    chown postgres:postgres "${DATA_VOLUME_MOUNTPOINT}/latest-lsn-checkpoint"
-    chmod 0300 "${DATA_VOLUME_MOUNTPOINT}/latest-lsn-checkpoint"
+  if [ ! -f "${LSN_CHECKPOINT_FILE_PATH}" ]; then
+    echo -n "0/0" > "${LSN_CHECKPOINT_FILE_PATH}"
+    touch "${LSN_CHECKPOINT_FILE_PATH}.previous"
+    chown postgres:postgres "${LSN_CHECKPOINT_FILE_PATH}"
+
+    chmod 0300 "${LSN_CHECKPOINT_FILE_PATH}"
   fi
 }
 
+function push_lsn_checkpoint_file {
```

Comment:
Good point, going to move LSN handling fully into adminmgr - better for it to be self-contained rather than handled in a handful of other ways. Would simplify the event listener considerably as well